### PR TITLE
add a few words about removing team AWS accounts

### PR DESCRIPTION
**Pull request by szd55gds**
**Why**
The process to remove unwanted AWS accounts was not obvious.

**What**
Adding a few words to explain the outline process.

### CODE REVIEW

File: source/documentation/automate-on-call-interruptible.md [hunk 1]
```diff
@@ -256,3 +256,9 @@ Relevant gds-cli account names:
 
 Main article: [GDS AWS Account Management
 Service](https://re-team-manual.cloudapps.digital/gds-aws-account-management-service.html)
+
+### Deleting AWS (team) Accounts no longer required
```
Comment (philandstuff) [inline source/documentation/automate-on-call-interruptible.md]: I'm not sure what `(team)` means here. Can we get rid of it or reword it?

File: source/documentation/automate-on-call-interruptible.md [hunk 2]
```diff
@@ -256,3 +256,9 @@ Relevant gds-cli account names:
...
+
+If the account points to the "aws-root-accounts" email address, then GDS staff who have access to the safe and membership of the 'aws-root-accounts' google group can remove the account.
```
Comment (philandstuff) [inline source/documentation/automate-on-call-interruptible.md]:
```suggestion
If the account uses the `aws-root-accounts@digital.cabinet-office.gov.uk` email address, then GDS staff who have access to the safe and membership of the `aws-root-accounts` google group can remove the account.
```

File: source/documentation/automate-on-call-interruptible.md [hunk 3]
```diff
@@ -256,3 +256,9 @@ Relevant gds-cli account names:
...
+
+If it does not point to the root account email address, the originating team should remove.
```
Comment (philandstuff) [inline source/documentation/automate-on-call-interruptible.md]:
```suggestion
If it uses a different email address, the account can only be removed by someone with access to that email address.
```

File: source/documentation/automate-on-call-interruptible.md (hunk 1 above)
Comment (szd55gds) [reply source/documentation/automate-on-call-interruptible.md]: Yes, I agree. Came from use of the word on the Request an AWS account page. Removed.

File: source/documentation/automate-on-call-interruptible.md [hunk 4]
```diff
@@ -256,3 +256,9 @@ Relevant gds-cli account names:
 
 Main article: [GDS AWS Account Management
 Service](https://re-team-manual.cloudapps.digital/gds-aws-account-management-service.html)
+
+### Deleting AWS Accounts no longer required
+
+If the account uses the `aws-root-accounts@digital.cabinet-office.gov.uk` email address, then GDS staff who have access to the safe and membership of the `aws-root-accounts` google group can remove the account.
+
+If it uses a different email address, the account can only be removed by someone with access to that email address.
```
Comment (Krenair) [inline source/documentation/automate-on-call-interruptible.md]:
```suggestion
If it uses a different email address, the account can only be removed by someone with access to that email address and associated MFA secret.
```

File: source/documentation/automate-on-call-interruptible.md [hunk 5]
```diff
@@ -256,3 +256,9 @@ Relevant gds-cli account names:
 
 Main article: [GDS AWS Account Management
 Service](https://re-team-manual.cloudapps.digital/gds-aws-account-management-service.html)
+
```
Comment (Krenair) [inline source/documentation/automate-on-call-interruptible.md]: should rename the header on line 253 to be like 'AWS Accounts'

**Pull request merged**